recipes/trollius/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "trollius" %}
+{% set version = "2.1" %}
+{% set sha256 = "cd165d9d9578d3aadbf26e71a88d9a70c4e444792a5628dc9af255f62030cc90" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - six
```

Comment:
Is `six` required to build the recipe? Also, since there is a c-extension, should `toolchain` be included at least for windows?